scripts/network.gd

```diff
@@ -1,8 +1,10 @@
 extends Node
 
-signal ws_received_message
+signal message_received(String)
 
-var _client = WebSocketClient.new()
+var _client := WebSocketClient.new()
+# Array<Dictionary>
+var _pending_messages: Array = []
```

Comment:
You can annotate the type as follows:
`var _pending_messages: Array[Dictionary] = []`